openff/toolkit/utils/rdkit_wrapper.py

```diff
@@ -1688,6 +1688,9 @@ def to_rdkit(cls, molecule, aromaticity_model=DEFAULT_AROMATICITY_MODEL):
             elif atom.stereochemistry == "R":
                 rdatom.SetChiralTag(Chem.CHI_TETRAHEDRAL_CCW)
 
+            # Stop rdkit from adding implicit hydrogens
+            rdatom.SetNoImplicit(True)
```

Comment:
I think I'd like @j-wags blessing too, but I give that explanation in the commit message to a0263f9. In short, we are already guaranteed to explicitly represent all our hydrogens, so its important to tell rdkit not to infer the existence of any other hydrogens. Without this change, converting a radical species to rdkit changes the molecule's identity. There are probably other related undiscovered bugs too.

Reply:
Thanks for the detailed explanation in that commit. I'm on board with this solution given your explanation, the fact that the tests are passing, our general lack of support/interest in radical species, and the likelihood that there are bigger problems with handling radicals. Until they're a key part of our force fields, I don't think it's worth losing sleep over them.

Given that Jeff is somewhat priority paralyzed at the moment I think it would be best to merge this in its current state. Unlikely as it may be, we could still catch breaking behavior before a next release. But please add a note to the "Behavior changed" section summarizing this change.

Reply:
Thanks for looping me in. I think this is a good call -- On the surface this is exactly what we want (OFFMols ALWAYS have all atoms explicit), and I can't think of any cases where this would have unintended effects. So this looks good to me, thanks for catching it @Yoshanuikabundi!